images/diagnostics/scripts/openshift-network-debug

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
unofficial shell coding style guidelines: https://github.com/openshift/origin/pull/6598/files

should be /bin/bash, not /bin/sh